pull_request_template.md

```diff
@@ -1,48 +1,26 @@
-<!--
-## Requirements
-
-- Filling out the template is required. Any pull request that does not include enough information to be efficiently reviewed may be rejected.
-- Please consider which branch this is to be submitted against. This will normally be the development branch, so please ask your reviewer if you think it needs to go somewhere else. 
--->
-
 ## Type
-
 - [ ] Bug fix
-- [ ] New feature
-- [ ] Improvement to existing feature
-
-## Related JIRA Issue(s)
-<!--_Please provide the URL(s) for any JIRA issues related to this PR._-->
-
-## Importance
-<!-- Please explain briefly if this PR specifically needs more importance and also if there is any due date by which it needs to be merged. -->
+- [ ] New feature / Improvement
+- [ ] Housekeeping
```

Comment:
What is housekeeping? I do use the `Improvement to exisiting feature` and i think it something good to know and to have as separate since they involved 2 different kind of work. Maybe the wording isn't right. Something like "Update existing feature" . An example is "Removing disabled props from radiogroup", this was kind of an update to existing feature.

Another category that we could have is package update since we often do these and they are quite specific changes.

Reply:
That's done